### PR TITLE
Math magicians full website

### PR DESCRIPTION
**Pull request by melkamu12**
> # Math magicians: full website
> In this PR, I added some functionality.
> 
> ## Activities done
> * [ ] Create a website consisting of 3 pages: Home, Calculator, and Quote.
> * [ ] Design general layout should match with the given wireframes:

> # Checklist:
> * [ ]  Follows the style guidelines of this project
> * [ ]  There is no linters error in my code
> * [ ]  follow gitflow

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        fringilla ex vitae sapien sagittis, eu porttitor nisi finibus. Nunc ut
+        metus eu mi scelerisque fermentum. Duis sagittis ultricies semper. Sed
+        pulvinar, tortor vel volutpat aliquam, nisi lorem eleifend neque, in
+        malesuada dolor nisi ut magna. Nullam cursus mi in tellus malesuada
+        rhoncus. Proin a diam eleifend, dictum massa at, congue est. Phasellus
+        auctor lectus ipsum, eu efficitur metus eleifend non. Fusce feugiat
+        semper dolor ac malesuada. Nunc sodales, orci in feugiat tincidunt,
+        tellus tellus suscipit velit, eget tincidunt eros turpis non tortor.
+        Ipsum.
+      </p>
```
Comment (souad988) [inline src/components/Home.js]: Good job by creating this view 👍🏼 , but please use your own text to make your work look professional it is not good to deploy your app with auto generated text

File: src/components/Root.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import MainNavigation from './mainNavigation';
+import classes from '../Root.module.css';
+
+function RootLayout() {
+  return (
+    <>
+      <MainNavigation />
+      <main className={classes.content}>
+        <Outlet />
+      </main>
```
Comment (souad988) [inline src/components/Root.js]: Well done here as well I want just to mention that the content here is not verticaly centered even with basic styling its very important to correctly align your element in the page.

Capture d’écran du 2023-07-19 11-13-35

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @melkamu12 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

242722065-db428b0c-8dfc-49c4-a156-42f227e32868


## Highlights
Use of react router routes
Good PR description
Navigation links displayed in all pages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @melkamu12 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**